pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Consumer.java

```diff
@@ -145,6 +146,19 @@ public String consumerName() {
         return consumerName;
     }
 
+    void notifyConsumerGroupChange(long activeConsumerId) {
+        if (cnx.getRemoteEndpointProtocolVersion() < ProtocolVersion.v11.getNumber()) {
```

Comment:
I was thinking here that we could have more descriptive methods in `Commands` class to group all these feature checks in a single place. Eg: 

```java
if (!Commands.peerSupportsActiveConsumerListener(cnx)) {
    return;
}
...
```

(I know in other places we're checking the version like this, and we should also change separately from this PR)

Reply:
One thing we need to be careful here is that C++ code gets compiled and automatically will present itself with the latest version v11 (or whatever). 

In this case, we should make sure that C++ client can at least discard the notification